### ISSUE TITLE
[Multi-ESX] VM's belonging to the same tenant are not visible across ESXs

### ISSUE THREAD

**Issue by ashahi1** [vmware/docker-volume-vsphere#1127]
Steps:
1. Initialized MultiNode config db on both the ESXs.
2. Created a tenant from ESX-1.
3. Added vm to the tenant from ESX-1.
4. Ran ```admin cli vm-group ls``` from ESX-1 and ESX-2.
VM_list field is empty for ```admin cli vm-group ls``` from ESX-2.

-------------------

Steps and their output are as follows:

1. Ran ```config init``` on both the ESXs.

```
ESX:1

[root@sc-rdops-vm06-dhcp-163-208:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py config init --datastore=sharedVmfs-0
Creating new DB at /vmfs/volumes/sharedVmfs-0/dockvols/vmdkops_config.db
Creating a symlink to /vmfs/volumes/sharedVmfs-0/dockvols/vmdkops_config.db at /etc/vmware/vmdkops/auth-db

ESX-2: 
[root@sc-rdops-vm06-dhcp-161-190:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py config init --datastore=sharedVmfs-0
Creating a symlink to /vmfs/volumes/sharedVmfs-0/dockvols/vmdkops_config.db at /etc/vmware/vmdkops/auth-db

```

2. Config information from both the ESXs:
``` 
ESX-1:
[root@sc-rdops-vm06-dhcp-163-208:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py status
=== Service:
Version: 0.12.9e2d6ad-0.0.1
Status: Running
Pid: 71898
Port: 1019
LogConfigFile: /etc/vmware/vmdkops/log_config.json
LogFile: /var/log/vmware/vmdk_ops.log
LogLevel: INFO
=== Authorization Config DB:
DB_LocalPath: /etc/vmware/vmdkops/auth-db
DB_Mode: MultiNode (local symlink pointing to shared DB)
DB_SharedLocation: /vmfs/volumes/sharedVmfs-0/dockvols/vmdkops_config.db
[root@sc-rdops-vm06-dhcp-163-208:~]

ESX-2:
[root@sc-rdops-vm06-dhcp-161-190:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py status
=== Service:
Version: 0.12.9e2d6ad-0.0.1
Status: Running
Pid: 108504
Port: 1019
LogConfigFile: /etc/vmware/vmdkops/log_config.json
LogFile: /var/log/vmware/vmdk_ops.log
LogLevel: INFO
=== Authorization Config DB:
DB_SharedLocation: /vmfs/volumes/sharedVmfs-0/dockvols/vmdkops_config.db
DB_LocalPath: /etc/vmware/vmdkops/auth-db
DB_Mode: MultiNode (local symlink pointing to shared DB)
[root@sc-rdops-vm06-dhcp-161-190:~]
```

3. Created a tenant from ESX-1
```
[root@sc-rdops-vm06-dhcp-163-208:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group create --name=vmgroup1
vm-group create succeeded
[root@sc-rdops-vm06-dhcp-163-208:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group ls
Uuid                                  Name      Description                        Default_datastore  VM_list
------------------------------------  --------  ---------------------------------  -----------------  -------
11111111-1111-1111-1111-111111111111  _DEFAULT  New description of default tenant
ed00d8a9-3709-4391-b7ed-333f0b5d41b1  vmgroup1

```
4. Added a vm to the tenant
```
[root@sc-rdops-vm06-dhcp-163-208:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group vm add --name=vmgroup1 --vm-list=photon-VM0.3
vm-group vm add succeeded
[root@sc-rdops-vm06-dhcp-163-208:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group ls
Uuid                                  Name      Description                        Default_datastore  VM_list
------------------------------------  --------  ---------------------------------  -----------------  ------------
11111111-1111-1111-1111-111111111111  _DEFAULT  New description of default tenant
ed00d8a9-3709-4391-b7ed-333f0b5d41b1  vmgroup1                                                        photon-VM0.3

```
5. Did vm-group ls from ESX-2 - tenant name is correct but VM_list is blank.
```
[root@sc-rdops-vm06-dhcp-161-190:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group ls
Uuid                                  Name      Description                        Default_datastore  VM_list
------------------------------------  --------  ---------------------------------  -----------------  -------
11111111-1111-1111-1111-111111111111  _DEFAULT  New description of default tenant
ed00d8a9-3709-4391-b7ed-333f0b5d41b1  vmgroup1

```

Logs:
[vmdk_ops_esx1.txt](https://github.com/vmware/docker-volume-vsphere/files/884876/vmdk_ops_esx1.txt)
[vmdk_ops_esx2.txt](https://github.com/vmware/docker-volume-vsphere/files/884875/vmdk_ops_esx2.txt)

**Comment by msterin** [comment]
Related to #1065

**Comment by pshahzeb** [comment]
Fixed through #1151
